### DISCUSSION TITLE
A Simpler Supabase-Type Front-End

### DISCUSSION THREAD

**Discussion by kubeworkz** [prest/prest#790] [Admin Frontend] [answered]
I'm looking to build a simpler front end for postgres with prest. First off, to your knowledge, has this been done? If not, I'm looking to build a simple user JWT authentication, and once authenticated, create a new project (basically a new DB) and produce a pg connection string. That's it. As far as I can tell, there's no open source project that does this anywhere. If anyone is interested in a project like this please let me know. Cheers, Dave

**Comment by arxdsilva** [comment, marked as answer]
Hi @kubeworkz , we started a while ago this project: https://github.com/prest/prestd-buildui

Althought It has not been maintained, If you'd like it would be a good starter point.

**Comment by kubeworkz** [comment]
Thanks, I'll check it out
